packages/terra-core-docs/src/terra-dev-site/doc/dropdown-button/About.1.doc.mdx

```diff
@@ -37,6 +37,11 @@ This component requires the following peer dependencies be installed in your app
 
 <!-- AUTO-GENERATED-CONTENT:END -->
 
+## Implementation Notes:
+The Form-Field component must be composed inside the [Base][1] component with a locale in order for it to load the correct translation strings.
+
+[1]: https://engineering.cerner.com/terra-core/components/terra-base/base/base
```

Comment:
this should point to application-base and we need to update react-intl as well on peer dependency table

Reply:
Updated URL to application base and already react-intl is there in peer dependency table.

<img width="822" alt="image" src="https://github.com/cerner/terra-core/assets/126655256/01be3864-ddb5-454c-b365-caa9c89549cd">